kyo-data/shared/src/main/scala/kyo/internal/ForSome.scala

```diff
@@ -6,7 +6,7 @@ type ForSome[F[_]] = ForSome.Type[F]
 object ForSome:
     class Unwrap[F[_], A](val unwrap: F[A]) extends AnyVal
 
-    opaque type Type[F[_]] <: Unwrap[F, ?] = Unwrap[F, ?]
```

Comment:
@road21 the compiler was crashing and removing `opaque` here fixed it

Reply:
https://github.com/scala/scala3/issues/23137